scripts/oncall-email/oncall.ml

```diff
@@ -66,106 +51,96 @@ let error j s =
   failwith s
 
 (* combinators *)
-let (>>) o fld =
+let ( >> ) o fld =
   match o with
   | J.Object xs ->
       let o = List.assoc fld xs in
       o
   | _ -> failwith (spf "could not find %s" fld)
 
-let jstring j =
-  match j with
-  | J.String s -> s
-  | _ -> error j "not a string"
+let jstring j = match j with J.String s -> s | _ -> error j "not a string"
```

Comment:
This kind of reformatting doesn't look very nice...

Reply:
Just to be more specific, I don't think that making as much code fit into an N-char line is the best formatting strategy.

Reply:
> Also we need to do those things in the weekend when everybody pushed their PRs.

yes, of course. This is a preview.

Reply:
Fixed with `break-cases = all` in the `.ocamlformat` file.